src/lib/database/entities/GuildEntity.ts

```diff
@@ -310,15 +279,15 @@ export class GuildEntity extends BaseEntity {
 	@Column('smallint', { name: 'selfmod.attachments.hardAction', default: 0 })
 	public selfmodAttachmentsHardAction = 0;
 
-	@ConfigurableKey({ dashboardOnly: true, description: LanguageKeys.Settings.DashboardOnlyKey })
-	@Column('integer', { name: 'selfmod.attachments.hardActionDuration', nullable: true })
+	@ConfigurableKey({ dashboardOnly: true, description: LanguageKeys.Settings.DashboardOnlyKey, minimum: Time.Second, maximum: Time.Year * 5 })
```

Comment:
Check didn't have a minimum
And `Time.Year * 5` is not the same as `1000`

Reply:
```sql
@Check(/* sql */ `"selfmod.attachments.hardActionDuration" >= 1000`)
```

Here the minimum is set to Time.Second, which is `1000`.